README.md

```diff
@@ -27,9 +27,32 @@ Then go to `http://localhost:8000/`
 
 ## Run tests
 
-```
-npm run test
-```
+- Unit tests
+
+  ```sh
+  npm run test:unit
+  ```
+
+- Integration tests
+
+  ```sh
+  npm run test:integration
+  ```
+
+- E2E tests
+
+  ```sh
+  npm install
+  npm run e2e:install
+
+  npm run clean
+  npm run build:web
+  npm run build:examples
+  npm run e2e:prepare-server
+
+  npm run test:e2e
+  npm run e2e:show-report
```

Comment:
I'll leave it with the following then:
```
npm install
npm run e2e:install
npm run build:web

npm run e2e:prepare-server
npm run test:e2e
npm run e2e:show-report
```

I'd like to have separate script `e2e:prepare-server` because we are not always want to run tests along side with the preparation.